KeY does not accept local files

## Description

When starting KeY from command line with a file arguments w/o prefixed explicit directory, KeY throws an exception.

## Reproducible
  
always

### Steps to reproduce 

Create a file a.key in a directory and then invoke KeY with it as a cli argument.

Like:

    echo '\problem { true }' > a.key
    ./gradlew installDist
    ./scripts/key --auto a.key

It should prove a.key, but if fails like:
```
Proof loading failedde.uka.ilkd.key.proof.init.ProofInputException: Cannot invoke "java.nio.file.Path.toUri()" because the return value of "java.nio.file.Path.getParent()" is null
	at de.uka.ilkd.key.proof.io.KeYFile.readIncludes(KeYFile.java:226)
	at de.uka.ilkd.key.proof.init.ProblemInitializer.readJava(ProblemInitializer.java:236)
	at de.uka.ilkd.key.proof.init.ProblemInitializer.prepare(ProblemInitializer.java:526)
	at de.uka.ilkd.key.proof.init.ProblemInitializer.prepare(ProblemInitializer.java:451)
	at de.uka.ilkd.key.proof.io.AbstractProblemLoader.createInitConfig(AbstractProblemLoader.java:547)
	at de.uka.ilkd.key.proof.io.AbstractProblemLoader.loadEnvironment(AbstractProblemLoader.java:309)
	at de.uka.ilkd.key.proof.io.AbstractProblemLoader.load(AbstractProblemLoader.java:273)
	at de.uka.ilkd.key.proof.io.ProblemLoader.doWork(ProblemLoader.java:75)
	at de.uka.ilkd.key.proof.io.ProblemLoader.runSynchronously(ProblemLoader.java:58)
	at de.uka.ilkd.key.ui.ConsoleUserInterfaceControl.loadProblem(ConsoleUserInterfaceControl.java:208)
	at de.uka.ilkd.key.core.Main.loadCommandLineFiles(Main.java:229)
	at de.uka.ilkd.key.core.Main.main(Main.java:197)
Caused by: java.lang.NullPointerException: Cannot invoke "java.nio.file.Path.toUri()" because the return value of "java.nio.file.Path.getParent()" is null
	at de.uka.ilkd.key.proof.io.KeYFile.readIncludes(KeYFile.java:222)
	... 11 common frames omitted
```

### Cause

Bisecting the problem revealed the cause to be 4e5cd04aa7a18caaa3f3a911400a1268587778c5.
Plausible since it claims to introduces java.nio over java.io.

### Note

I thought we had solved this problem already, but now it resurfaces??

---

* Commit: 9d0379c47bf9eaff35068fe68a087e7ee0a508c6

<!-- END OF BUG TEMPLATE -->

--- a/key.core/src/main/java/de/uka/ilkd/key/proof/io/KeYFile.java
+++ b/key.core/src/main/java/de/uka/ilkd/key/proof/io/KeYFile.java
@@ -220,7 +220,7 @@ public class KeYFile implements EnvInput {
             try {
                 KeyAst.File ctx = getParseContext();
                 // weigl: fix #3721, absolute path is required to solve relative filenames.
-                var absPath = file.file().toAbsolutePath().getParent();
+                Path absPath = file.file().toAbsolutePath().getParent();
                 includes = ctx.getIncludes(absPath.toUri().toURL());
             } catch (ParseCancellationException e) {
                 throw new ParseCancellationException(e);
@@ -242,7 +242,7 @@ public class KeYFile implements EnvInput {
         Path bootClassPathFile = Paths.get(bootClassPath);
         if (!bootClassPathFile.isAbsolute()) {
             // convert to absolute by resolving against the parent path of the parsed file
-            var parentDirectory = file.file().getParent();
+            Path parentDirectory = file.file().getParent();
             bootClassPathFile = parentDirectory.resolve(bootClassPath);
         }
 
@@ -261,7 +261,7 @@ public class KeYFile implements EnvInput {
     @Override
     public @NonNull List<Path> readClassPath() {
         ProblemInformation pi = getProblemInformation();
-        var parentDirectory = file.file().getParent();
+        Path parentDirectory = file.file().getParent();
         List<Path> fileList = new ArrayList<>();
         for (String cp : pi.getClasspath()) {
             if (cp == null) {
@@ -282,10 +282,10 @@ public class KeYFile implements EnvInput {
         ProblemInformation pi = getProblemInformation();
         String javaPath = pi.getJavaSource();
         if (javaPath != null) {
-            var absFile = Paths.get(javaPath);
+            Path absFile = Paths.get(javaPath);
             if (!absFile.isAbsolute()) {
                 // convert to absolute by resolving against the parent path of the parsed file
-                var parent = file.file().getParent();
+                Path parent = file.file().getParent();
                 absFile = parent.resolve(javaPath);
             }
             if (!Files.exists(absFile)) {

